ExampleCodes/Basic/HeatEquation_EX4_C/Exec/GNUmakefile

```diff
@@ -0,0 +1,37 @@
+# AMREX_HOME defines the directory in which we will find all the AMReX code.
+AMREX_HOME ?= ../../../../../amrex
+
+DEBUG        = FALSE
+USE_MPI      = FALSE
+USE_OMP      = FALSE
+COMP         = gnu
+DIM          = 3
+USE_RPATH    = TRUE
+
+include $(AMREX_HOME)/Tools/GNUMake/Make.defs
+
+include ../Source/Make.package
+VPATH_LOCATIONS  += ../Source
+INCLUDE_LOCATIONS += ./Source
+
+CPPFLAGS += -Werror -Wsuggest-override
```

Comment:
I am asking because I am not sure: can we take this line out or is it serving some other purpose? For me, it turns several warnings into errors, preventing the compile.